podvm: Enable se image build for rhel with and without --no-verify

ANJANA-A-R-K opened the pull request:
Made changes to build RHEL SE enabled podvm image with an extra Arg of SE_VERIFY=true

<!--
If this is a bug fix, make sure your description includes "Fixes: #xxxx", or
"Closes: #xxxx"

Please provide the following information:
-->
**- Description of the problem which is fixed/What is the use case**
This PR introduces an additional feature to verify and download required certificates for secure environments when the SE_VERIFY tag is set to true

**- What I did**
I added a conditional block that checks if the SE_VERIFY environment variable is set to true. If so, it performs the following actions:

Downloads the IBM Z Host Key Signing Certificate (ibm-z-host-key-signing-gen2.crt).

Downloads the DigiCert CA certificate (DigiCertCA.crt).

Downloads the IBM Z Host Key CRL (ibm-z-host-key-gen2.crl).

**- How to verify it**
Built and deploy the pod VM image.
```
[root@bastion-rr-initdata ~]# oc get pods
NAME          READY   STATUS    RESTARTS   AGE
busybox-new   1/1     Running   0          41s
```

**- Description for the changelog**
Added SE_VERIFY flag to enable downloading and verifying security certificates for secure prod environments.

ANJANA-A-R-K committed:
podvm: Enable se image build for rhel with and without --no-verify

Made changes to build RHEL SE enabled podvm image with an extra Arg of SE_VERIFY=true

Signed-off-by: ANJANA-A-R-K <anjana.a.r.k1@ibm.com>

bpradipt commented on config/peerpods/podvm/libvirt-podvm-image-cm.yaml:
```diff
@@ -25,6 +25,12 @@ data:
   PODVM_IMAGE_URI: "" # eg: oci::quay.io/openshift_sandboxed_containers/libvirt-podvm-image:latest::/image/podvm.qcow2
+
+  # Name of PodVM qcow2 to be built.
+  IMAGE_NAME: "podvm-image" 
```
Is `IMAGE_NAME` used in libvirt handler ?

ANJANA-A-R-K replied:
yes it is being referenced in cloud-api-adaptor https://github.com/confidential-containers/cloud-api-adaptor/blob/0fb6afebc06909a2a6da62f389bb3abe5eda05f2/src/cloud-api-adaptor/podvm/Makefile.inc#L70
without this parameter the image creation jobs fails.
`Makefile.inc:77: *** Failed to derive an image name. Explicitly define IMAGE_NAME.  Stop.`

bpradipt replied:
nit: For other providers we are setting IMAGE_NAME in the handler code. You may want to check that to be sure..

bpradipt reviewed (approved):
lgtm

snir911 reviewed (approved):
LGTM, also if it would make sense maybe it should be considered to have the certificates statically

littlejawa commented:
/ok-to-test

littlejawa commented:
/ok-to-test

ANJANA-A-R-K committed:
podvm: Enable se image build for rhel with IMAGE_NAME in handler script
Made changes to build RHEL SE enabled podvm image IMAGE_NAME in handler script

Signed-off-by: ANJANA-A-R-K <anjana.a.r.k1@ibm.com>

littlejawa commented:
/retest
